include/ibamr/CIBStochasticStokesSolver.h

```diff
@@ -0,0 +1,235 @@
+// Filename: CIBStaggeredStokesSolver.h
```

Comment:
```suggestion
// Filename: CIBStochasticStaggeredStokesSolver.h
```
Also, do we really need a whole new solver here, or could we just massage the RHS in (a subclass of) the existing `CIBStaggeredStokesSolver`?

Reply:
If it is technically possible to do this but too much work for now, please add a TODO comment.

If there really needs to be a different solver, please address this in the class documentation.